unicodetools/src/main/java/org/unicode/props/IndexUnicodeProperties.java

```diff
@@ -722,9 +722,11 @@ class IndexUnicodeProperty extends UnicodeProperty.BaseProperty {
         @Override
         public boolean isTrivial() {
             return _getRawUnicodeMap().isEmpty()
-                    || _getRawUnicodeMap()
-                            .keySet(_getRawUnicodeMap().getValue(0))
-                            .equals(UnicodeSet.ALL_CODE_POINTS);
+                    || ((_getRawUnicodeMap().stringKeys() == null
+                                    || _getRawUnicodeMap().stringKeys().isEmpty())
```

Comment:
FYI: a UnicodeMap.hasStrings() function would be nice...

Reply:
Yes.